packages/asl-public-api/test/specs/projects.js

```diff
@@ -27,6 +28,15 @@ describe('/projects', () => {
       });
   });
 
+  it('returns project licenses to add course which is approved for higher education or training purposes', async () => {
+    const res = await openAPI.validateGet(
+      this.api,
+      `/establishment/${ids.establishments.trainingEstablishment}/projects/cat-e`);
```

Comment:
the tool needs to look up the route by its path template. I couldn't find a way to infer the route from the url, so you'll need to provide the path template and a parameter object.

```suggestion
    const res = await openAPI.validateGet(
      this.api,
      '/establishment/{establishmentId}/projects/cat-e',
      {establishmentId: ids.establishments.trainingEstablishment}
    );
```